samples/sample_capture_assistant.py

```diff
@@ -0,0 +1,25 @@
+"""Capture Assistant sample."""
+import datetime
+import zivid
+from zivid.captureassistant import AmbientLightFrequency, SuggestSettingsParameters
+
+
+def _main():
+    app = zivid.Application()
+    camera = app.connect_camera()
+
+    suggest_settings_parameters = SuggestSettingsParameters(
+        budget=datetime.timedelta(milliseconds=1200),
+        frequency=AmbientLightFrequency.hz50,
+    )
+
+    suggested_settings = zivid.captureassistant.suggest_settings(
+        camera, suggest_settings_parameters
+    )
+
+    frame = zivid.hdr.capture(camera, suggested_settings)
```

Comment:
You should use the context manager idiom, like in the other samples (see `sample_capture_hdr.py`).
It doesn't really matter in a simple example like this, but we should do it for consistency.

```
with zivid.hdr.capture(camera, suggested_settings) as as hdr_frame:
    hdr_frame.save("result.zdf"
```

Reply:
Done in fixup.